blog/_posts/DelSquared_RGSoC_blogpost.md

```diff
@@ -0,0 +1,47 @@
+﻿
+---

```

Comment:
Hi @arushi019 ! No need of the blank line at the start of an article and blank spaces at the start of parameters. It changes the format. Please test the blog post for markdown format. You can test it in some online tool also if you don't want to run the site locally.